tests/scaffoldings/addons_env_double/prod.yaml

```
@@ -0,0 +1,29 @@
+version: "2.1"
+services:
+  odoo:
+    build:
+      context: ./
+      args:
+        COMPILE: "false"
+        ODOO_VERSION: $ODOO_MINOR
+        WITHOUT_DEMO: "false"
+    tty: true
+    depends_on:
+      - db
+    environment:
+      PYTHONOPTIMIZE: ""
+      UNACCENT: "false"
```

I don't think you really need 2 separate docker-compose files if you add this here:

```suggestion
    environment:
      DOODBA_ENVIRONMENT: "$DOODBA_ENVIRONMENT"
      PYTHONOPTIMIZE: ""
      UNACCENT: "false"
```

Of course, you must add that env variable to the test execution env too.